### ISSUE TITLE
[RyuJIT/arm32] Assertion failed 'foundStructArg'

### ISSUE THREAD

**Issue by mskvortsov** [dotnet/coreclr#11781]
```
$ COMPlus_AltJit=BringUpTest:Main \
gdb --args clr-debug/corerun \
tests-release/JIT/Directed/TypedReference/TypedReference/TypedReference.exe
...
#9  0xb0c6c1cc in assertAbort (why=0xb0f1dcd5 "foundStructArg", file=0xb0f1cef7 "/home/mskvortsov/git/coreclr/src/jit/morph.cpp", line=4727) at /home/mskvortsov/git/coreclr/src/jit/error.cpp:287
#10 0xb0d4bda6 in Compiler::fgMorphMultiregStructArgs (this=0x93f10, call=0x97bc8) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:4727
#11 0xb0d45654 in Compiler::fgMorphArgs (this=0x93f10, call=0x97bc8) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:4423
#12 0xb0d58608 in Compiler::fgMorphCall (this=0x93f10, call=0x97bc8) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:8362
#13 0xb0d435d0 in Compiler::fgMorphTree (this=0x93f10, tree=0x97bc8, mac=0x0) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:15005
#14 0xb0d5d64e in Compiler::fgMorphStmts (this=0x93f10, block=0x973c8, mult=0xbeffc337, lnot=0xbeffc336, loadw=0xbeffc335) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:15804
#15 0xb0d5e00a in Compiler::fgMorphBlocks (this=0x93f10) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:16089
#16 0xb0d601cc in Compiler::fgMorph (this=0x93f10) at /home/mskvortsov/git/coreclr/src/jit/morph.cpp:17291
```
```
GC/API/GC/Collect_Forced_1
GC/API/GC/Collect_Forced_2
GC/API/GC/Collect_Forced_3
JIT/Directed/TypedReference/TypedReference
JIT/IL_Conformance/Old/Conformance_Base/refs
JIT/Methodical/flowgraph/dev10_bug679008/fgloop
JIT/Methodical/flowgraph/dev10_bug679053/regionLive
JIT/Methodical/refany/array1
JIT/Methodical/refany/array2
JIT/Methodical/refany/_dbgvirtcall
JIT/Methodical/refany/format
JIT/Methodical/refany/_il_dbgarray1
JIT/Methodical/refany/_il_dbgarray2
JIT/Methodical/refany/_il_dbgarray3
JIT/Methodical/refany/_il_dbgformat
JIT/Methodical/refany/_il_dbgindcall
JIT/Methodical/refany/_il_dbglcs
JIT/Methodical/refany/_il_dbglongsig
JIT/Methodical/refany/_il_dbgshortsig
JIT/Methodical/refany/_il_relarray1
JIT/Methodical/refany/_il_relarray2
JIT/Methodical/refany/_il_relarray3
JIT/Methodical/refany/_il_relformat
JIT/Methodical/refany/_il_relindcall
JIT/Methodical/refany/_il_rellcs
JIT/Methodical/refany/_il_rellongsig
JIT/Methodical/refany/_il_relshortsig
JIT/Methodical/refany/lcs
JIT/Methodical/refany/_relvirtcall
JIT/Methodical/refany/_speed_dbgvirtcall
JIT/Methodical/refany/_speed_relvirtcall
JIT/Methodical/refany/virtcall
JIT/Methodical/VT/etc/_il_dbgnested
JIT/Methodical/VT/etc/_il_relnested
JIT/Methodical/VT/etc/_speed_relgc_nested
JIT/Methodical/VT/etc/_speed_relnested
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_I8
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_I
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_R4
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_R8
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_U2
JIT/Methodical/xxobj/ldobj/_il_dbgldobj_V
JIT/Methodical/xxobj/ldobj/_il_relldobj_I8
JIT/Methodical/xxobj/ldobj/_il_relldobj_I
JIT/Methodical/xxobj/ldobj/_il_relldobj_R4
JIT/Methodical/xxobj/ldobj/_il_relldobj_R8
JIT/Methodical/xxobj/ldobj/_il_relldobj_U2
JIT/Methodical/xxobj/ldobj/_il_relldobj_V
JIT/Methodical/xxobj/operand/_speed_relrefanyval
JIT/Regression/CLR-x86-JIT/V1-M12-Beta2/b51575/b51575
JIT/Regression/CLR-x86-JIT/V1-M12-Beta2/b52593/b52593
JIT/Regression/CLR-x86-JIT/V1-M12-Beta2/b52733/b52733
JIT/Regression/CLR-x86-JIT/V1-M12-Beta2/b52840/b52840
JIT/Regression/CLR-x86-JIT/V1-M13-RTM/b92066/b92066
JIT/Regression/CLR-x86-JIT/V2.0-RTM/b487364/b487364
JIT/Regression/CLR-x86-JIT/V2.0-RTM/b487372/b487372
```

cc @dotnet/arm32-contrib

**Comment by hseok-oh** [comment]
I started this. 
@BruceForstall Please assign this to me.

**Comment by hseok-oh** [comment]
Fixed by #12568 
@mskvortsov Please close this issue
